docs/pages/deployment/deploy.md

```diff
@@ -50,20 +50,18 @@ And finally to deploy all required LunaSec resources to your AWS account, run:
 lunasec deploy
 ```
 
-This command will create the file `aws_resources.yaml`. This file will contain the ARNs for the resources that have been deployed to AWS.
-
-Among the resources in the file will be a URL which points to the Tokenizer Backend:
-
-```yaml
-tokenizer:
-  gateway_endpoint: https://<gateway_id>.execute-api.us-west-2.amazonaws.com/prod/
+Let's find the URL that the tokenizer is deployed to:
+```shell
+$ lunasec resources
+Tokenizer URL: https://asdf.execute-api.us-west-2.amazonaws.com/prod/
```

Comment:
This guys pretty static right?  what are the chances someone is going to want this output as JSON for doing programmatic stuff?  With lerna, you can switch any output to JSON mode for machine-readable use.